dotcom-rendering/src/web/components/EnhancePinnedPost.importable.tsx

```diff
@@ -12,14 +12,24 @@ const pinnedPostCheckBox: Element | null = !isServer
 	? window.document.querySelector('input[name=pinned-post-checkbox]')
 	: null;
 
+const pinnedPostContent: Element | null = !isServer
+	? window.document.querySelector('#collapsible-body')
+	: null;
+
 /**
- * hide show more button on pinned post
+ * hide show more button and overlay on pinned post
  */
 function hideShowMore() {
 	const pinnedPostButton = document.querySelector<HTMLElement>(
 		'#pinned-post-button',
 	);
-	if (pinnedPostButton) pinnedPostButton.style.display = 'none';
+	const pinnedPostOverlay = document.querySelector<HTMLElement>(
+		'#pinned-post-overlay',
+	);
+	if (pinnedPostButton && pinnedPostOverlay) {
+		pinnedPostButton.style.display = 'none';
+		pinnedPostOverlay.style.display = 'none';
+	}
```

Comment:
Thanks Max! I did try to do this first of all but it throws the following exception:
`The left-hand side of an assignment expression may not be an optional property access.ts(2779)`. Because of this I've left the if check in but I've separated into two checks, one for each element.